devimint/src/util.rs

```diff
@@ -13,6 +14,24 @@ use tracing::{debug, warn};
 
 use super::*;
 
+pub fn parse_map(s: &str) -> anyhow::Result<BTreeMap<String, String>> {
+    let mut map = BTreeMap::new();
+
+    if s.is_empty() {
+        return Ok(map);
+    }
+
+    for pair in s.split(',') {
+        let parts: Vec<&str> = pair.split('=').collect();
+        if parts.len() == 2 {
+            map.insert(parts[0].to_string(), parts[1].to_string());
+        } else {
+            return Err(format_err!("Invalid pair in map: {}", pair));
+        }
+    }
+    Ok(map)
+}
+
```

Comment:
Can't you import this duplicated function from a common dependency of `fedimintd` and `devimint`?

Reply:
Me and me were debating it while implementing and reviewing it yesterday.

I don't know if we should bloat public interface of `fedimint-core` with every utility function that we happen to use in multiple places.

We probably ... want some `fedimint-internal` or `fedimint-utils-internal` that we could specifically keep low key, so external users don't use it. But even then we will have deal with it's semver, possibly by just incrementing major release like there's no tomorrow.